R/utils.R

```diff
@@ -377,3 +377,33 @@ get_n_samples.array <- function(x){
   stopifnot(is_3d_array(x))
   return(dim(x)[1])
 }
+
+#' Code to use the hesim package inline. Not directly called by the user.
+#' @param ... arguments
+#' @examples
+#' library(Rcpp)
+#' sourceCpp(code="
+#' // [[Rcpp::depends(hesim)]]
+#' // [[Rcpp::depends(RcppArmadillo)]]
+#' #include <hesim/stats/distributions.h>
+#' // [[Rcpp::export]]
+#' double test_inline_gengamma(double mu, double sigma, double Q) {
```

Comment:
It would be good to add something like this as a unit test.

Reply:
Looks like you had to remove the test?

Reply:
Yes: testing inline seems to require an installed package (using package/include/...) and does not work with a source tree (using package/inst/include/...). There may be some test magic to solve this. The test is also slow (2-4 seconds).

Note that I have found the inline facility very helpful for prototyping the cCTSTM implementations.